src/realmModels/Observation.js

```diff
@@ -192,7 +192,7 @@ class Observation extends Realm.Object {
   }
 
   static projectUri = obs => {
-    const photo = obs.observation_photos[0];
+    const photo = obs?.observation_photos?.[0];
```

Comment:
unrelated bug, but this was giving me undefined errors when viewing nearby projects on android